docs/reference/modules/settings.md

```diff
@@ -0,0 +1,127 @@
+---
+
+# `@apostrophecms/settings`
+
+**Alias:** `apos.settings`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module governs the Personal Settings menu that allows users to change their preferences. Apostrophe projects can configure this menu through `@apostrophecms/settings`.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`subforms`](#subforms) | Object | Each object defines an input field in the Personal Settings menu. |
+| [`groups`](#groups) | Object | Like the property of the same name in the `fields` setting, this option organizes the subforms on tabs in the Personal Settings menu. |
+
+<AposCodeBlock>
+
+``` javascript
+module.exports = {
+  options: {
+    subforms: {
+      displayName: {
+        fields: [ 'displayName' ],
+        previewComponent: 'SettingsDisplayNamePreview',
+        protection: true,
+        reload: true
+      },
+      name: {
+        label: 'Name',
+        fields: [ 'firstName', 'lastName' ]
+        // This preview will be automatically generated.
+        // preview: '{{ firstName }} {{ lastName }}'
+      },
+      adminLocale: {
+        fields: [ 'adminLocale' ]
+      },
+      changePassword: {
+        // This will have `protection: true` automatically.
+        fields: [ 'password' ]
+      }
+    },
+
+    groups: {
+      account: {
+        label: 'Account',
+        subforms: [ 'name', 'displayName', 'changePassword' ]
+      },
+      preferences: {
+        label: 'Preferences',
+        subforms: [ 'adminLocale' ]
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:cation>
+  /modules/@apostrophecms/settings/index.js
+</template>
+</AposCOdeBlock>
+### `subforms`
+
+The `subforms` option takes an object of named objects.  Each individual object has a required `fields` property that takes an array of strings that are the names of existing schema fields in the `@apostrophecms/user` settings. By default, the `password` and `adminLocale` fields are available for addition. Note that the `adminLocale` field needs to be enabled by [setting the `adminLocales` option](/reference/modules/i18n.html) of the `@apostrophecms/i18n` module.
+
+In addition to the required `fields` property, each object can take five additional optional properties.
+
+The `label` property takes a string to display to the user in both preview and edit mode. If not supplied, Apostrophe will use the label for the first schema passed to the `fields` property. 
+
+The `help` property takes a field to display to the user in preview mode. If this property is present, it will replace any component added through the option `previewComponent` property. The `previewComponent` property takes the name of a Vue component that has been added to your project through any `modules/my-custom-module/ui/apos/components` folder. The string passed to this property should not contain the file extension. [See below](#previewcomponent) for further information about the component structure.
```

Comment:
"Help" is text (or an i18n key) that appears when in preview mode (when not editing that subform). If it is not present then the i18n key "preview" is used, and this is typically an i18next key that receives placeholders like `{{ firstName }} {{ lastName}}`, leveraging i18next as a built-in templating system and allowing different locales to put them in different orders as appropriate. If neither is present then `previewComponent` is rendered via Vue. If `previewComponent` doesn't exist either, then the fields of the subform are concatenated as space-separated strings for preview purposes.

TODO: I'm not entirely sure why both `help` and `preview` need to exist and I've asked Miro.

Reply:
I rewrote/rearranged this after talking with Miro a bit. `previewComponent`>`help`>`preview`>automatic assignment